src/ocean/util/container/map/Map.d

```diff
@@ -1228,6 +1228,14 @@ unittest
     test_val!(long);
 }
 
+unittest
+{
+    // ensure opIn works with const keys
+    auto map = new StandardKeyHashingMap!(int, mstring)(5);
+    auto v = "something" in map;
+    test!("is")(v, null);
+}
```

Comment:
Sadly no, and I don't know if this can be fixed trivially:

```
 Error: function ocean.util.container.map.Map.Map!(string, string).Map.opIn_r (const(string) key) is not callable using argument types (const(char)[])
```

Problem is that `istring` requires strictly `istring` as input, not some `cstring`. We know that in this case it doesn't matter and could redefine it as `opIn_r ( in Unqual!(ElementTypeOf!(T))[] )` but as base class is generic for any element/key types, it will quickly get messy :/